Feature/edit bulk upload rows

boveus opened the pull request:
Hey @ewilloughby this feature adds data tables to the bulk upload data rows screen.  It also adds an ajax request that occurs when a user edits one of the fields.  There is a little bit of logic in the controller, but I think it makes sense to keep there for now (creating a presenter seems like it would be overkill for 4 or so lines).  Let me know what you think.

ewilloughby commented on app/assets/javascripts/sirenia/bulk_upload_show.js:
```diff
@@ -0,0 +1,42 @@
+$(document).ready( function () {
+    $('.bulkupload-table').DataTable({
+        "dom": 'C<"clear">lfrtip'
+    })
```
Based on feedback about paging in Hyrax, I strongly suspect they'll want to see 100 rows (or maybe more, depending on the size of bulk-upload jobs) at once without having to click next.  That would be added here:

```javascript
$('.bulkupload-table').DataTable({
        "dom": 'C<"clear">lfrtip',
        "pageLength": 100
    })
```

ewilloughby commented on app/views/sirenia/bulk_uploads/show.html.erb:
```diff
@@ -8,24 +8,28 @@
 
-<table class="table table-striped table-hover">
+<h5> Bulk Upload Data Rows </h5>
+
```
To make the table responsive, surround it with a div with class .table-responsive.  This will make the table controls a bit more responsive along with adding a horizontal scrollbar to the table itself when needed.
```html
<div class="table-responsive">
<table>...
</table>
</div>
```

boveus commented:
Hey @ewilloughby I just changed up a few of the settings for the Data Tables.  Instead of having the responsiveness set in the div I added that key value pair to the JS.  I noticed an issue with the scrolling where the header would disappear, so I reworked the look and feel of the page a bit.  We can get feedback from Kathryn and Jeremy and modify it as needed,  but I think these changes makes it a little easier to use.
